api/handler/notification.go

```diff
@@ -18,11 +19,18 @@ func notification(router chi.Router) {
 }
 
 func getNotification(writer http.ResponseWriter, request *http.Request) {
-	start, err := strconv.ParseInt(request.URL.Query().Get("start"), 10, 64)
+	urlValues, err := url.ParseQuery(request.URL.RawQuery)
+	if err != nil {
+		render.Render(writer, request, api.ErrorInvalidRequest(err)) //nolint
```

Comment:
nolint (((((

Reply:
`Render` returns an error, `nolint` is needed so that the linter does not swear that the error is ignored